code/modules/urist/modules/wfc/core/turf.dm

```diff
@@ -0,0 +1,40 @@
+/turf
+	var/wfc_overwritable = FALSE
+
+	// list of refs to objects to delete on overwrite
+	var/list/associated_wfc_overwritables = null
+
+
+/turf/procgen
+	icon = 'icons/urist/turf/uristturf.dmi'
+	icon_state = "tramfloor"
+
+	wfc_overwritable = TRUE
+
+
+/turf/procgen/wfc
+
+
+/proc/WfcChangeTurf(var/turf/trg, var/new_type)
+	if(isnull(new_type))
+		return
+
+	if(!ispath(new_type))
+		return
+
+	if(!ispath(new_type, /turf))
+		return
```

Comment:
Seems like some redundant checks here. `ispath(new_type, /turf)` would return false if it is not a path, or if it were null.
```suggestion
	if(!ispath(new_type, /turf))
		return
```

Reply:
I left those in as placeholders for more specific debug logging ("oh, it failed because it's a null/not a turf/an instance")